### DISCUSSION TITLE
Domain kerberos and opossite behavior related to dns

### DISCUSSION THREAD

**Discussion by 0xdeface** [FreeRDP/FreeRDP#10228] [Q&A]
Hello 123.123.0.123 is my DC, my lapton have not join to our domain and when i connect **by ip** from my office domain network to target server i get a stuck.
But when i connect with different dns like 1.1.1.1 or from home my connection works great.
 xfreerdp /v:123.123.0.123 /u:'I*****r' /p:'****' /sec:nla
also i noticed in console i have  [kerberos_AcquireCredentialsHandleA]: krb5glue_get_init_creds (Cannot find KDC for realm 
i bet it related to me. What i should do for connect with my dns DC, maybe xfreerdp have some appropriate parameter?

**Comment by akallabeth** [comment]
1. check your local `krb5.conf` file, it most likely has some invalid entries in there
1. connect by domain name instead of IP
2. disable kerberos with `/auth-pkg-list:!kerberos`
